render/container.go

```diff
@@ -264,6 +264,11 @@ func MapContainer2IP(m report.Node) []string {
 			if !ok {
 				continue
 			}
+			// loopback addresses are shared among all namespaces
+			// so we can't use them to attribute connections to a container
+			if report.IsLoopback(addr) {
+				continue
+			}
```

Comment:
This is only for short-lived connections (obtained through conntrack and not proc) in which there won't be a PID (and by extension we can't either provide a networking namespace ID either).

In those cases `127.0.0.1:8080` could be the listening socket of **any** container with a loopback connection.